### ISSUE TITLE
Module can't be installed from the registry

### ISSUE THREAD

**Issue by nightmareartist** [terraform-google-modules/terraform-google-sql-db#515]
### TL;DR

It seems that the new version of this module hasn't been published in the Terraform registry: https://registry.terraform.io/modules/GoogleCloudPlatform/sql-db/google/latest making it impossible to install v17.0.0 in the usual way.

### Expected behavior

When using `source  = "GoogleCloudPlatform/sql-db/google//modules/postgresql"` the latest version of the module should be installed, in this case `17.0.0`

### Observed behavior

Terraform complains that there is no version `17.0.0` in the registry and the registry itself has `16.1.0` as the latest version available.

### Terraform Configuration

```hcl
Simply set the version of the module to `17.0.0` and run `terraform plan`. It will fail right away.
```

### Terraform Version

```sh
Terraform v1.5.7
on darwin_arm64
```

**Comment by a-zen** [comment]
I'm having the same issue.

**Comment by mirobertod** [comment]
Same here

**Comment by g-awmalik** [comment]
Seems like the registry stopped pulling the tags. I've updated it and v17 should be available for download now. Will monitor this moving forward.

**Comment by g-awmalik** [comment]
Seems like there's a problem on the TF registry side. Currently here have https://registry.terraform.io/modules/terraform-google-modules/sql-db/google/latest that is pulling the latest versions. Investigating why https://registry.terraform.io/modules/GoogleCloudPlatform/sql-db/google/latest stopped at v16.1.0.

Report this with Hashicorp: https://github.com/hashicorp/terraform/issues/34186

**Comment by g-awmalik** [comment]
This is now resolved.
